### PR TITLE
feat: add foundry in SDK

### PR DESCRIPTION
**Pull request by ChiHaoLu**
I write a foundry template and introduction for Kakarot Sepolia.

### CODE REVIEW

**Review by Eikix — COMMENTED** [review]
Hey! Awesome addition, thanks! can you update it for our new testnet migration? Kakarot Starknet Sepolia

**Comment by ChiHaoLu** [comment]
Yes, I will! 

But it is embarrassed that I cannot claim faucet in the discord bot haha...

**Comment by Eikix** [comment]
> Yes, I will!
> 
> But it is embarrassed that I cannot claim faucet in the discord bot haha...

Ah yes, sorry and thank you. The bot is currently down as we migrate to Starknet Sepolia

**Comment by ChiHaoLu** [comment]
@Eikix I give a new version for the latest network!

File: docs/starknet/ecosystem/sdks/foundry.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+### Setup
+
+You can declare a alias in `foundry.toml`:
+```
+[profile.default]
+src = "src"
+out = "out"
+libs = ["lib"]
+evm_version = 'shanghai' # Kakarot Sepolia supports `PUSH0`
```
Comment (Eikix) [inline docs/starknet/ecosystem/sdks/foundry.md]: we even support TLOAD / TSTORE -> dencun

**Comment by Eikix** [comment]
> @Eikix I give a new version for the latest network!

Does this work locally:)?
Can we check it too @swetshaw

**Comment by ChiHaoLu** [comment]
> Does this work locally:)? Can we check it too @swetshaw

Yes! I have deployed and verified contract with this Foundry on the Kakarot StarkNet Sepolia!

**Comment by Eikix** [comment]
> > Does this work locally:)? Can we check it too @swetshaw
> 
> Yes! I have deployed and verified contract with this Foundry on the Kakarot StarkNet Sepolia!

You're awesome! Thank you so much

**Comment by ChiHaoLu** [comment]
I add the compile with `cancun` in the `foundry.toml` setting. And deploy it successfully. [Contract Link](https://sepolia.kakarotscan.org/address/0x1d1688f3CF762fda6E53cb97522487F96edaF1b8/contract/920637907288165/code)

**Comment by ChiHaoLu** [comment]
Hello @Eikix , does this PR look alright to you?

**Review by Eikix — APPROVED** [review]

**Pull request merged**